src/main/java/org/killbill/billing/plugin/analytics/dao/factory/BusinessFactoryBase.java

```diff
@@ -369,7 +368,7 @@ protected Plan getPlanFromInvoiceItem(final InvoiceItem invoiceItem, final Local
             final Catalog catalog = getCatalog(context);
             return catalog.findPlan(invoiceItem.getPlanName(), invoiceItem.getStartDate().toDateTimeAtStartOfDay(), subscriptionStartDate.toDateTimeAtStartOfDay());
         } catch (CatalogApiException e) {
-            logService.log(LogService.LOG_INFO, "Unable to retrieve plan for invoice item " + invoiceItem.getId(), e);
```

Comment:
I think it was `INFO` level because of https://github.com/killbill/killbill/issues/784, but I'm fine changing it to `WARN`.

Reply:
Good catch. But I think it makes sense as well -;)